include/grpc/impl/channel_arg_names.h

```diff
@@ -400,6 +400,8 @@
   "grpc.max_allowed_incoming_connections"
 /** Configure per-channel or per-server stats plugins. */
 #define GRPC_ARG_EXPERIMENTAL_STATS_PLUGINS "grpc.experimental.stats_plugins"
+/** If non-zero, allow security frames to be sent and received. */
+#define GRPC_ARG_SECURITY_FRAME_ALLOWED "grpc.security_frame_allowed"
 /** \} */
```

Comment:
Just curious. Why do we need a channel arg for this.

Reply:
We don't usually  make comments about internal behavior in the public code. I can add a comment internally regarding this difference.